HACKING.txt

```diff
@@ -1,15 +1,17 @@
 Hacking on Substance D
 ======================
 
-Here are some guidelines about hacking on Substance D.
+The following are guidelines and instructions for hacking on and with Substance
+D.
 
-Using a Development Checkout
-----------------------------
+Use a development checkout
+--------------------------
```

Comment:
- This is a WIP. More to come, not ready to merge yet. We ran into testing issues, and this was to see whether it would pass on Travis-CI and to share its results in #pyramid on IRC.
- Title case is for titles of books or chapters, not necessarily for headings at various levels. It's acceptable to use either title case or sentence case for headings. In the Pyramid docs, I've found numerous bad examples of "How To Title Case A Section" (where prepositions and articles are incorrectly capitalized), so I opted for the easier to not screw up form of "How to sentence case a section". It is also less "shouty", easier to read, and modern since headings can be displayed in a bold font face. Historically title case also served the purpose to distinguish headlines from the narrative, but it's becoming less common to do so.